extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -46,6 +46,14 @@ import {parseJson} from '../../../src/json';
 import {user} from '../../../src/log';
 import {waitForBodyPromise} from '../../../src/dom';
 
+/**
+ * @enum {number}
+ */
+export const GEO_IN_GROUP = {
+  NOT_DEFINED: 0,
```

Comment:
I learned my lesson last time to not use `0` as a enum value. the check for `if (!value)` is error prone, can we start with 1 instead : )

Reply:
done